tests/unit/reaction/test_combinatorics.py

```diff
@@ -206,3 +208,23 @@ def test_generate_permutations(
     assert spin_permutations[1][2][1] == +1
     assert spin_permutations[2][0][1] == +1
     assert spin_permutations[3][0][1] == +1
+
+
+def test_perform_external_edge_identical_particle_combinatorics(
+    y_to_d0_d0bar_pi0_pi0,
```

Comment:
You dont really need a full solution graph here. So creating some artificial dummy graph would do it.